tests/remote-target-lists/top-1k-crates-2022-6-27.txt

```diff
@@ -0,0 +1,621 @@
+http://github.com/SSheldon/rust-block
```

Comment:
How hard is it to generate this list? I am tempted to drop this file for now. I can see us integrating the top 100 to our CI, but I'm guessing the top 1k is not in our near future.

Reply:
OK. Removed with a663ec603c2dbcdcfaa096dade0313ec3d74dcf6